index.html

```diff
@@ -1,14 +1,80 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <head>
```

Comment:
looks like there are too many heads?

Reply:
thx for comment. do you mean head or header? I only see one head

Reply:
sorry - that was my mistake - it is all okay, I interpreted something the wrong way! all lgtm 👍